.github/workflows/ci.yaml

```diff
@@ -103,3 +111,28 @@ jobs:
           env:
             DJANGO_SETTINGS_MODULE: hedera.test_settings
           run: python manage.py test
+
+  js_tests:
+    name: Running javascript unit tests
+    runs-on: ubuntu-latest
+
+    steps:
+        - uses: actions/checkout@v2
+
+        - uses: actions/setup-node@v2
+          with:
+            node-version: '14'
+
+        - name: Cache node modules
+          uses: actions/cache@v2
+          with:
+            path: ~/.npm
+            key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+            restore-keys: |
+              ${{ runner.os }}-node-
+
+        - name: Install Static Dependences
+          run: npm i --include=dev
```

Comment:
Is it my imagination or is the `--include` flag an [undocumented](https://docs.npmjs.com/cli/v8/commands/npm-install) feature of the CLI? I was just curious as I couldn't find that listed anywhere.

Reply:
@arthurian I found a snippet in the `omit` section of the docs https://docs.npmjs.com/cli/v8/commands/npm-install#omit

Reply:
Good find @Tanvez -- I used the [penultimate step of the linting job](https://github.com/Hedera-Lang-Learn/hedera/pull/379/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR49-R50) as a template for this, @arthurian, so didn't really consider the flag, as it was working; I'm guessing it is explicit about/ensures dev dependencies, even if `NODE_ENV=production`.